src/sentry/static/sentry/app/components/assistant/helper.jsx

```diff
@@ -54,6 +55,11 @@ const AssistantHelper = createReactClass({
     });
   },
 
+  handleCueDismiss() {
```

Comment:
This is now identical to `handleDismiss` in guide drawer, so we can pass it as a prop to guide drawer instead of duplicating it.